app/src/main/java/com/main/mealplanner/UI/DetailsFragment.kt

```diff
@@ -14,16 +14,20 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import com.main.mealplanner.MainViewModel
 import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.RecyclerView
+import com.main.mealplanner.MainActivity
 import com.main.mealplanner.R
 import com.main.mealplanner.dto.RecipeDetails
+import com.main.mealplanner.dto.RecipeHeader
 import com.squareup.picasso.Picasso
 import kotlinx.android.synthetic.main.details_fragment.*
 import kotlinx.android.synthetic.main.main_fragment.*
+
 import java.util.ArrayList
 
 open class DetailsFragment: Fragment(){
     internal lateinit var viewModel: MainViewModel
-    private var recipeDtl = RecipeDetails()
+    private var recipeDetails = RecipeDetails()
```

Comment:
when going through the code recipeDtl dint scream out that it was RecipeDetails with having to look were it being implemented.

remove unused imports, didn't see "in the work code" here so I choose to remove them

Reply:
Yes.  While it feels like you're saving some typing by abbreviating, you'll lose much more time later, trying to remember what the abbreviation is.

Reply:
Good point. At my co-op we always abbreviate "detail" to "dtl" just due to the nature of our ERP system that we use, but I can see how that is not intuitive to other people.